sp800-63c/sec5_strength.md

```diff
@@ -70,11 +70,11 @@ All assertions SHOULD use audience restriction techniques to allow an RP to reco
 
 #### 5.2.5. Pairwise Pseudonymous Identifiers
 
-In some circumstances, it is desirable to prevent the subscriber's account at the CSP from being linked through one or more RPs through use of a common identifier. Pairwise pseudonymous subject identifiers SHALL be used within the assertions generated by the CSP for the RP, and the CSP SHALL generate a different identifier for each RP.
+In some circumstances, it is desirable to prevent the subscriber's account at the CSP from being linked through one or more RPs through use of a common identifier. Pairwise pseudonymous subject identifiers SHALL be used within the assertions generated by the CSP for the RP, and the CSP SHALL generate a different identifier for each RP or integration with an RP, as appropriate.
 
-When unique pseudonymous identifiers are used with RPs along side of identity attribute bundles, it may still be possible for multiple colluding RPs to fully identify and correlate a subscriber across systems using these attributes. Even though a different pseudonymous identifier was used at each RP, the RPs are still able to compare other identifying attributes such as names, email addresses, physical addresses, and phone numbers, among other common attributes. In such cases, the privacy risk is similar to the subscriber not having used a pseudonymous identifier at all.
+When unique pseudonymous identifiers are used with RPs along side of identity attribute bundles, it may still be possible for multiple colluding RPs to fully identify and correlate a subscriber across systems using these attributes. Privacy policies, therefore, may prohibit such correlation, but pairwise pseudonymous identifiers can increase effectiveness of these policies by increasing the administrative effort in managing the attribute correlation. 
 
-Note that in a brokered federation, a CSP cannot issue a pseudonymous identifier for each RP because it is unaware which RP is being accessed. In this case the broker, as a proxy CSP, may wish to generate pseudonymous identifiers to prevent privacy-violating collusion among RPs. Depending on the protocol, the broker may often need to map such identifiers back to the associated identifiers from upstream CSPs in order to allow the identity protocol to function.
+Note that in a distributed federation model, the distributing party that is acting as a proxy CSP may, depending on the protocol, need to map the pairwise pseudonymous identifiers back to the associated identifiers from upstream CSPs in order to allow the identity protocol to function.
```

Comment:
This change is incorrect and deletes important information about the limitations of brokered systems with regard to pseudonymous identifiers. This has statement nothing to do with distributed systems.